files/anitya_cron.py

```diff
@@ -93,7 +94,13 @@ def main(debug, feed):
         projects = list(projects_by_feed(session))
         session.commit()
     else:
-        projects = db.Project.all(session)
+        # Get all projects, that are ready for check
+        projects = session.query(
```

Comment:
You can use ``db.Project.query...`` to get a query, rather than ``session.query(db.Project)``, FYI